libs/execution_context/ebpf_program.c

```diff
@@ -53,9 +53,12 @@ typedef struct _ebpf_program
     const ebpf_extension_data_t* info_extension_provider_data;
     // Program type specific helper function count.
     uint32_t provider_helper_function_count;
```

Comment:
```suggestion
    uint32_t program_type_specific_helper_function_count;
```

Reply:
Changed to `program_specific_helper_function_count`